.buildkite/scripts/dra.sh

```diff
@@ -103,14 +107,15 @@ dra() {
   fi
 }
 
-dra "snapshot" "$dra_command"
-if [[ "${DRA_BRANCH}" != "main" && "${DRA_BRANCH}" != "8.x" ]]; then
-  echo "DRA_BRANCH is neither 'main' nor '8.x'"
-  dra "staging" "$dra_command"
-fi
-
-## Exception for main branch as requested by the Release Team.
-if [[ "${DRA_BRANCH}" == "main" ]]; then
-  echo "Exception form main branch with the qualifier alpha1"
-  dra "staging" "$dra_command"
+if [[ "${TYPE}" == "snapshot" ]]; then
+  # qualifier is not needed for snapshots, let's unset it.
+  dra "${TYPE}" "$dra_command" ""
+else
+  ## Exception for main branch as requested by the Release Team.
+  #if [[ "${DRA_BRANCH}" != "main" && "${DRA_BRANCH}" != "8.x" ]]; then
```

Comment:
can you remove this and the comment below please

Reply:
Sure, I kept it only because I assumed the Release Team might request to revert the `main`/`staging` support.